_people/Nishad_Muhammed.markdown

```diff
@@ -0,0 +1,23 @@
+---
+name: Nishad Muhammed
+lastname: Muhammed
+group: graduate_students
+position: Phd Candidate
+institution: Washington State University
+advisor: Matthew Duez
+phone: 
+email: nishad.muhammed@wsu.edu
+address: 
+specialties: Numerical Relativity, Hypermassive Neutron Stars, Post-Merger Compact Systems. 
+---
+
+##### Research Profile
+
+I work on [SpEC](https://www.black-holes.org/for-researchers/spec). I have studied dynamical stability of hypermassive neutron stars produced after binary neutron star mergers. Currently I am studying energy release from neutrino-antineutrino annihilation in accretion disks formed by binary neutron star mergers and collapsed stars, which could be responsible for gamma-ray bursts.
+
+##### Biography
+
+I am a PhD candidate at Washington State University, working under the supervision of Dr. Matthew Duez. I received my BS in Physics from University of Dhaka. Before starting my current position, I received my MS working with Dr. Gaurav Khanna in Gravitational Wave Physics at Umass Dartmouth. For more details, check out my [website] (https://nishadmhd1313.wixsite.com/nishadmuhammed) 
```

Comment:
Remove space between square brackets and round brackets
```suggestion
I am a PhD candidate at Washington State University, working under the supervision of Dr. Matthew Duez. I received my BS in Physics from University of Dhaka. Before starting my current position, I received my MS working with Dr. Gaurav Khanna in Gravitational Wave Physics at Umass Dartmouth. For more details, check out my [website](https://nishadmhd1313.wixsite.com/nishadmuhammed) 
```

Reply:
FYI we also support putting your web site in the metadata at the top, i.e. a line of the form
```yaml
website: https://nishadmhd1313.wixsite.com/nishadmuhammed
```